index.bs

```diff
@@ -1074,6 +1085,221 @@ spec: webidl
     </p>
   </section>
 </section>
+<section>
+  <h2 id="automation">
+    Automation
+  </h2>
+  <p>
+    For the purposes of user-agent automation and application testing, this
+    document defines the following <a>errors</a>, algorithms, and <a>extension
+    commands</a> for the [[WebDriver]] specification.
+  </p>
+
+  The <dfn>no such request</dfn> <a>error</a> has a [=response/status=] of 404
```

Comment:
WebDriver's extension mechanism does not currently provide an explicit means of specifying new "errors", so I am making this up as we go. I'll follow up with Simon in this pull request's main comment thread.